### PR TITLE
Do misc things

### PR DESCRIPTION
**Pull request by Sean-Tedrow-LB**
**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: sdk/rust/src/account/account_create_transaction.rs [hunk 1]
```diff
@@ -80,6 +83,10 @@ pub struct AccountCreateTransactionData {
     ///
     pub max_automatic_token_associations: u16,
 
+    pub alias_key: Option<PublicKey>,
+
+    pub alias_evm_address: Option<[u8; 20]>,
```
Comment (mehcode) [inline sdk/rust/src/account/account_create_transaction.rs]: `alias_evm_address` is now just `evm_address` right?

Comment (Sean-Tedrow-LB) [reply sdk/rust/src/account/account_create_transaction.rs]: Yep.  Good catch

File: sdk/rust/src/account/account_id.rs [hunk 2]
```diff
@@ -72,12 +77,48 @@ impl AccountId {
...
 
+    /// Create an `AccountId` from a solidity address.
+    pub fn from_solidity_address(address: &str) -> crate::Result<Self> {
+        let EntityId { shard, realm, num, checksum } = EntityId::from_solidity_address(address)?;
+
+        Ok(Self { shard, realm, num, alias: None, evm_address: None, checksum })
+    }
+
+    /// Accepts "0x___" or "___" Ethereum public address.
+    pub fn from_evm_address(address: &str) -> crate::Result<Self> {
```
Comment (izik1) [inline sdk/rust/src/account/account_id.rs]: https://github.com/hashgraph/hedera-sdk-reference/issues/71

we get to make `EvmAddress` public and then pass that around instead, yay.

File: sdk/rust/src/account/account_create_transaction.rs [hunk 3]
```diff
@@ -145,6 +162,24 @@ impl AccountCreateTransaction {
...
+    ///
+    /// If a transaction creates an account using an alias, any further crypto transfers to that alias will
+    /// simply be deposited in that account, without creating anything, and with no creation fee being charged.
+    pub fn alias_key(&mut self, key: PublicKey) -> &mut Self {
+        self.body.data.alias = Some(key);
+        self
+    }
+
+    /// The last 20 bytes of the keccak-256 hash of a ECDSA_SECP256K1 primitive key.
+    pub fn alias_evm_address(&mut self, evm_address: [u8; 20]) -> &mut Self {
```
Comment (izik1) [inline sdk/rust/src/account/account_create_transaction.rs]: https://github.com/hashgraph/hedera-sdk-reference/issues/69
@mehcode I pinged you on this just now elsewhere

File: sdk/rust/src/schedule/schedule_create_transaction.rs [hunk 4]
```diff
@@ -301,6 +302,13 @@ impl ToTransactionDataProtobuf for ScheduleCreateTransactionData {
                 transaction_body::Data::ContractCall(data) => {
                     Some(schedulable_transaction_body::Data::ContractCall(data))
                 }
+                // TODO: implement these
+                Data::NodeStakeUpdate(_) => {
+                    unimplemented!("NodeStakeUpdate has not been implemented")
```
Comment (izik1) [inline sdk/rust/src/schedule/schedule_create_transaction.rs:307]: `todo!` rather than `unimplemented!`

**Review by izik1 — DISMISSED** [review]

**Review by izik1 — DISMISSED** [review]

**Review by izik1 — APPROVED** [review]

**Comment by izik1** [comment]
CI appears stuck (isn't getting past "fetch swift"); tested swift 5.5 locally, everything passes, everything compiles, would be weirder if it didn't

**Pull request merged**